mongo_orchestration/replica_sets.py

```diff
@@ -434,6 +435,8 @@ def connection(self, hostname=None, read_preference=pymongo.ReadPreference.PRIMA
                     c = pymongo.MongoClient(
                         servers, socketTimeoutMS=20000,
                         w='majority', fsync=True, **self.kwargs)
+                    connected(c)
+                    c.admin.command('ismaster')
```

Comment:
Oh yes it is; fixed.

On Mon, Mar 9, 2015 at 3:25 PM, Luke Lovett <notifications@github.com>
wrote:

> In mongo_orchestration/replica_sets.py
> <https://github.com/10gen/mongo-orchestration/pull/178#discussion_r26066569>
> :
>
> > @@ -434,6 +435,8 @@ def connection(self, hostname=None, read_preference=pymongo.ReadPreference.PRIMA
> >                      c = pymongo.MongoClient(
> >                          servers, socketTimeoutMS=20000,
> >                          w='majority', fsync=True, **self.kwargs)
> > +                    connected(c)
> > +                    c.admin.command('ismaster')
>
> Is this redundant?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/10gen/mongo-orchestration/pull/178/files#r26066569>.
>